src/base_object.h

```diff
@@ -76,6 +76,7 @@ class BaseObject : public MemoryRetainer {
   // e.g. when the JS object used `MakeLazilyInitializedJSTemplate`.
   static inline void SetInternalFields(v8::Local<v8::Object> object,
                                        void* slot);
+  static inline bool IsBaseObject(v8::Local<v8::Object> object);
   static inline void TagNodeObject(v8::Local<v8::Object> object);
```

Comment:
Can we rename this method to be like `TagBaseObject` aligning with `IsBaseObject`?